### ISSUE TITLE
Error in new version 0.20.0 when creating apk in Windows 11 Pro

### ISSUE THREAD

**Issue by Ambassador-of-programming** [flet-dev/flet#2638]
Installed the new version pip install flet==0.20.0

I get an error when creating apk file:
```
flet build apk                                                                              
Creating Flutter bootstrap project...OK                                                                                                 
Customizing app icons and splash images...OK
Generating app icons...OK
Generating splash screens...OK
Packaging Python app...OK
Building .apk for Android...Resolving dependencies...
  ffi 2.1.0 (2.1.2 available)
  fl_chart 0.65.0 (0.66.2 available)
  flutter_lints 2.0.3 (3.0.1 available)
  flutter_native_splash 2.3.7 (2.3.11 available)
  js 0.6.7 (0.7.0 available)
  lints 2.1.1 (3.0.0 available)
  matcher 0.12.16 (0.12.16+1 available)
  material_color_utilities 0.5.0 (0.8.0 available)
  meta 1.10.0 (1.12.0 available)
  package_info_plus 4.2.0 (5.0.1 available)
  path 1.8.3 (1.9.0 available)
  petitparser 5.4.0 (6.0.2 available)
  test_api 0.6.1 (0.7.0 available)
  toml 0.14.0 (0.15.0 available)
  web 0.3.0 (0.5.0 available)
  web_socket_channel 2.4.0 (2.4.3 available)
  xml 6.3.0 (6.5.0 available)
Got dependencies!
17 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.

Running Gradle task 'assembleRelease'...
Warning: The plugin url_launcher_android requires Android SDK version 34.
For more information about build configuration, see
https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
Running Gradle task 'assembleRelease'...                           64,6s

One or more plugins require a higher Android SDK version.
Fix this issue by adding the following to C:\Users\pro\AppData\Local\Temp\flet_flutter_build_GWinlQeIdc\android\app\build.gradle:       
android {
  compileSdkVersion 34
  ...
}

C:\Users\pro\AppData\Local\Android\sdk\ndk\23.1.7779620\toolchains\llvm\prebuilt\windows-x86_64\bin\llvm-strip.exe: error:
'C:\Users\pro\AppData\Local\Temp\flet_flutter_build_GWinlQeIdc\build\serious_python_android\intermediates\merged_native_libs\release\out
\lib\armeabi-v7a\libpythonbundle.so': The file was not recognized as a valid object file

C:\Users\pro\AppData\Local\Android\sdk\ndk\23.1.7779620\toolchains\llvm\prebuilt\windows-x86_64\bin\llvm-strip.exe: error:
'C:\Users\pro\AppData\Local\Temp\flet_flutter_build_GWinlQeIdc\build\serious_python_android\intermediates\merged_native_libs\release\out
\lib\arm64-v8a\libpythonbundle.so': The file was not recognized as a valid object file

C:\Users\pro\AppData\Local\Android\sdk\ndk\23.1.7779620\toolchains\llvm\prebuilt\windows-x86_64\bin\llvm-strip.exe: error:
'C:\Users\pro\AppData\Local\Temp\flet_flutter_build_GWinlQeIdc\build\serious_python_android\intermediates\merged_native_libs\release\out
\lib\x86_64\libpythonbundle.so': The file was not recognized as a valid object file

lib/main.dart💯43: Error: The argument type 'List<dynamic>' can't be assigned to the parameter type 'List<Widget?
Function(CreateControlArgs)>?'.
 - 'List' is from 'dart:core'.
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart'
('../../../../dev/flutter/packages/flutter/lib/src/widgets/framework.dart').
 - 'CreateControlArgs' is from 'package:flet/src/control_factory.dart'
('../../Pub/Cache/hosted/pub.dev/flet-0.20.0/lib/src/control_factory.dart').
                  createControlFactories: createControlFactories
                                          ^
lib/main.dart:119:49: Error: The argument type 'List<dynamic>' can't be assigned to the parameter type 'List<Widget?
Function(CreateControlArgs)>?'.
 - 'List' is from 'dart:core'.
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart'
('../../../../dev/flutter/packages/flutter/lib/src/widgets/framework.dart').
 - 'CreateControlArgs' is from 'package:flet/src/control_factory.dart'
('../../Pub/Cache/hosted/pub.dev/flet-0.20.0/lib/src/control_factory.dart').
                        createControlFactories: createControlFactories
                                                ^
Target kernel_snapshot failed: Exception


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command 'C:\Users\pro\dev\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 3s
Gradle task assembleRelease failed with exit code 1

Error building Flet app - see the log of failed command above.
```
Operating system: Windows 11 Pro
Version: 23H2
OC build: 22631.3155

Installed Flet and created a `flet create scr` project
and then `cd scr` and `flet build apk`

**Comment by Ambassador-of-programming** [comment]
Everything worked successfully in version 0.19.0

**Comment by FeodorFitsner** [comment]
The issue has been fixed now.

**Comment by sumitkanoje** [comment]
@FeodorFitsner in which version the issue is fixed? 0.21.0 ?
